test/unit/driver-specs.js

```diff
@@ -244,7 +244,7 @@ describe('driver', () => {
       }).to.throw(/must include/);
       expect(() => {
         driver.validateDesiredCaps({platformName: 'Android', deviceName: 'device', browserName: 'Netscape Navigator'});
-      }).to.throw(/Netscape Navigator/);
+      }).to.throw(/must include/);
```

Comment:
Since `validateDesiredCaps` when passing in an unknown browser no longer fails with:
```
The desiredCapabilities object was not valid for the following reason(s): browserName Netscape Navigator is not included in the list.
```

Change the test to check for the error:
```
The desired capabilities must include either an app, package or browser
```